_posts/2020-12-22-ols.md

```diff
@@ -0,0 +1,76 @@
+
+## [Growing the Galaxy Community](https://openlifesci.org/ols-2/projects-participants/#growing-the-galaxy-community), leaded by [Beatriz Serrano-Solano](https://openlifesci.org/ols-2/projects-participants#beatrizserrano) and mentored by [Dave Clements](https://openlifesci.org/ols-2#tnabtaf)
+
+Galaxy is an open-source framework that enables scientists to perform computational biology analysis. A worldwide community supports the Galaxy project providing tools, workflows, computational resources and training materials in many different scientific areas.
+
+At the beginning of September 2020, Beatriz started a new position at the University of Freiburg to help with the community management within the European Galaxy team. European Galaxy community has grown both in number and fields, standing in need of coordination across several countries that need to work collaboratively to achieve common goals. Her new role includes the responsibility of coordinating the European Galaxy and scientific community as well as strengthening the relationship with the Australian community, helping build the Asian and African communities, and event organisation around Bioconda, BioContainers and Galaxy. With this project, she advanced her community building and project management skill while enhancing the connection with the communities outside Europe.
+
+You can watch her graduation presentation there:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/wv8_0iOMpGs?start=1639" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## [Creating a single pipeline for metagenome classification](https://openlifesci.org/ols-2/projects-participants/#creating-a-single-pipeline-for-metagenome-classification), leaded by [Muhammet Celik](https://openlifesci.org/ols-2/projects-participants#MuhammetCelik) and mentored by [Mallory Freeberg](https://openlifesci.org/ols-2#malloryfreeberg)
+
+Metagenomics has been increasingly becoming very important in studies of human and animal health. It has become clear that bacteria in and on our bodies are very significant. Thus there is a big boost in science society who are willing to study metagenomics. When we want to do downstream analysis in our microbial community, the first step is to find out about our community, what the community ultimately looks like.
+
+The idea of the project was to benchmark the state of the art tools (such as Kraken2, Centrifuge and CLARK) and create a single pipeline that will produce a single pdf file with cutoff-signifance-sensitivity values for each tool. Later we can set that pipeline on web-based system that would make the researchers easier for the scientists.
+
+The project started as a command-line tool. But after a meeting with Bérénice Batut, he decided to develop his pipeline in Galaxy. ***Add more details there***
```

Comment:
@macelik Here if you have few sentences to suggest would be awesome

Reply:
@bebatut The main reason for this was that this did not consider the compute need variability among users for their differing size of input data. In fact, many users interested in such an analysis would like not have the necessary compute resource to execute the pipeline. Further, there was the OS compatibility issue. So, instead of leaving this burden for the user to shoulder, Galaxy appeared to be better and more intuitive solution.